tests/e2e/test_grade_inquiry.py

```diff
@@ -76,6 +83,9 @@ def test_no_submission_grade_inquiry_panel(self):
 
      # STUDENT SUBMISSION TESTS
     def test_normal_submission_student_grade_inquiry_box(self):
+        # travis should not run this
+        if "TRAVIS_BUILD_DIR" in os.environ and os.environ["TRAVIS_BUILD_DIR"] is not None:
+            return
```

Comment:
Mark these tests as properly [skipped](https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures) by adding annotation `@unittest.skipUnless(os.environ.get('TRAVIS_BUILD_DIR') is None, "cannot run in Travis-CI")` so that we have a visual indicator that is what's going on here.